doc/api/debugger.md

```diff
@@ -145,6 +145,7 @@ debug>
 ### Information
 
 * `backtrace`, `bt` - Print backtrace of current execution frame
+* `frame(3)`, `f(3)` - Select stack frame #3 visible in the backtrace
```

Comment:
It doesn't take variables, it only takes number literals.  Also, this matches the `list` example further down.